packages/server/src/admin/super.ts

```diff
@@ -37,10 +37,7 @@ superAdminRouter.post(
     requireSuperAdmin();
     requireAsync(req);
 
-    await sendAsyncResponse(req, res, async () => {
-      await rebuildR4ValueSets();
-      await getSystemRepo().reindexResourceType('CodeSystem');
-    });
+    await sendAsyncResponse(req, res, async () => rebuildR4ValueSets());
```

Comment:
Why was the CodeSystem reindex removed?  I think this will be necessary to transition self-hosted customers to the new terminoloogy service implementation.

Reply:
I thought the terminology service results are built dynamically at query-time, so that was not necessary?

`reindexResourceType('CodeSystem')` is big and slow, so tests were timing out.  I was under the impression that it is not a necessary step after `rebuildR4ValueSets()`.

Reply:
No, I believe it is necessary to transition self-hosted customers.  How else will all the R4 CodeSystem resources get loaded into the `Coding` table?